src/scripts/imagehelper.js

```diff
@@ -3,7 +3,7 @@ define(["browser"], function (browser) {
 
     function getDeviceIcon(device) {
         var baseUrl = "assets/img/devices/";
-        switch (device.AppName) {
+        switch (device.AppName || device.Client) {
```

Comment:
the first one is correct

Reply:
Both are used within the web ui though. It’s not a matter of what clients are sending, there is different object types being passed to this function depending on where it’s being called. The section on the main dashboard that lists the current active devices is broke and always uses the default device icon. The dedicated device page works as expected though.

Reply:
Agh that's annoying, I guess we need this for now then. Why can't the endpoints just report the same values.